src/scene/scene.js

```diff
@@ -140,7 +140,7 @@ function Scene() {
         lastStaticPrepareWriteTime: 0,
         lastStaticPrepareTriAabbTime: 0,
         lastStaticPrepareCombineTime: 0,
-        updateShadersTime: 0
+        updateShadersTime: 0 // deprecated
```

Comment:
Just curious, but why is this now deprecated?

Reply:
it was only used around the functions I optimized, which take no time now .. so no need to report duration

Reply:
OK, fair enough. 🚀